content/posts/towards-krustlet-v1-cncf.md

```diff
@@ -0,0 +1,136 @@
+  of Kubernetes in [Rust][rust], a memory-safe systems programming langauge.
+
+Today we are extremely excited to announce that Krustlet was accepted in the
+[Cloud Native Computing Foundation (CNCF)][cncf] as a Sandbox project, alongside
+[Krator][krator], a Kubernetes Rust state machine operator framework written in
+Rust, as well as the first alpha release of Krustlet, [v1.0.0-alpha.1][alpha1].
+
+This marks the start of API and feature stability for Krustlet, and as we
+approach the final 1.0 release, of strong backwards compatibility guarantees.
+
+### The 1.0 roadmap
+
+Krustlet enables developers to build and run WebAssembly and WASI modules on
+Kubernetes. Today, you can take advantage of Kubernetes features such as pods,
+secrets, init containers, or volume mounts, and in the next couple of weeks, we
+will continue to stabilize the current API and feature set. Here are the
+specific guarantees for each type of release:
+
+- alpha: No backwards compatibility or API guarantees. However, we do not intend
+  to do any major breaking changes, but we reserve the right to should the need
+  arise
+- beta: Similar to alpha, we will not likely be breaking anything. The one
+  exception would be to solve any bugs or missing features found during beta
+  releases
+- RC: Backwards compatibility and API stability guarantees will go into effect.
+  Breaking changes will only be considered for showstopper bugs
+- 1.0.0+: No breaking changes will be allowed nor anything that changes
+  backwards compatibility. Any breaking changes will be deferred to a future 2.0
+  release
+
+> You can get started with Krustlet by following [the official
+> documentation][docs].
+
+### What does 1.0 even mean?
+
+"But wait," you ask, "how is this 1.0 when you are still missing things like
+full Kubernetes networking?" We chose to move Krustlet to 1.0 because using it
+as a Kubelet is now stable enough to actually _build real things_ on top of.
+WebAssembly is still rapidly evolving and so there are still rough edges and
+other problems to work through. It would be a disservice to the community to not
+stabilize what we have so other people can keep learning about and innovating
+with WebAssembly. Additionally, there are others in the Rust community who are
+building other types of Kubelets that have nothing to do with WebAssembly and we
+owe it to them to have a stable version on which to build their own projects.
+
+So, what does this being 1.0 _actually_ mean? First and foremost, it means
+stability and backwards compatibility guarantees, as mentioned above. This means
+that people can start to build applications built to run on Krustlet (or with
+tooling from Krustlet) without fear of a constantly evolving API. This includes
+ourselves, where we have plans to create a
+[WAGI](https://github.com/deislabs/wagi) Krustlet provider. It also means that
+the WebAssembly bits are to the point you can build _something_ with them. Not
```

Comment:
Should be `them -- not` (em-dash). This is because `Not everything...` is a dependent clause. Conversely, you can invert the clause order in the `Not everything` sentence: `You can build something real but...`. English is a mind-numbing language.